accounts/tests/utils/utils.py

```diff
@@ -0,0 +1,8 @@
+from accounts.models import User
+
+
+def create_staff_user():
+    test_user = User.objects.create(
+        email="user@test.com",
+    )
+    return test_user
```

Comment:
nit: we can just return this right away!
```suggestion
    return User.objects.create(
        email="user@test.com",
    )
```